tests/kafkatest/services/verifiable_consumer.py

```diff
@@ -123,7 +123,7 @@ class VerifiableConsumer(BackgroundThreadService):
     logs = {
         "verifiable_consumer_stdout": {
             "path": STDOUT_CAPTURE,
-            "collect_default": False},
```

Comment:
We only print the start and end offsets of message batches received at the moment. But eventually we'll probably have test cases which require seeing message payloads.